src/manifest/index.ts

```diff
@@ -118,6 +122,8 @@ export default class Manifest {
       assert(this.presentationLiveGap != null);
       assert(this.timeShiftBufferDepth != null);
     }
+
+    this.loadedAt = Date.now() / 1000;
```

Comment:
Weird to have the ``loadedAt`` variable defined like that in that data structure.

For me, the ``Manifest`` class just better structures a Manifest object given by ``net`` and provide methods to help its exploitation.

Here we add the notion that it should be instanciated (and only instanciated) just after loading it.